conduct.md

```diff
@@ -40,25 +40,34 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Be proactive about communicating.
+
+- Constructively criticize others.
+
+- Acknowledge others.
```

Comment:
Please provide some examples/descriptions of how you should acknowledge others -- similar to other comments.